extension-base/ctpAzureWrapper.js

```diff
@@ -0,0 +1,77 @@
+// Azure
+
+module.exports = {
+  ctpAzureWrapper: function(fn) {
+    return function(context, req) {
+      fn(req.body, ctpAzureResponse(context), context.log);
+    }
+  }
+}
+
+function ctpAzureResponse(context) {
+  return {
+    pass: passAzure(context),
+    errors: errorsAzure(context),
+    error: errorAzure(context),
+    updates: updatesAzure(context),
+    update: updateAzure(context),
+  }
+}
+
+function passAzure(context) {
```

Comment:
Depends on what Node version this runs on you could use arrow functions. [node.green](http://node.green/#ES2015-functions-arrow-functions) is a good reference to see what works where.

This would become

```js
const createPassAzure = ctx => () => {
   ctx.res = {
     status: 200,
     body: undefined
   };
   ctx.done();
}
```

Reply:
Looks like the three major ones (AWS, Azure, Google) are on 6 something. Thanks for the link 👍